proto/api/v1/plan.proto

```
@@ -0,0 +1,30 @@
+  rpc ListPlans(ListPlansRequest) returns (ListPlansResponse) {}
+}
+
+message ListPlansRequest {
+}
+
+message ListPlansResponse {
+  repeated Plan plans = 1;
+}
+
+enum FeatureSet {
+  BASIC = 0;
```

Buf (1.54.0)

Enum value name "BASIC" should be prefixed with "FEATURE_SET_".

(ENUM_VALUE_PREFIX)